internal/model/board.go

```diff
@@ -264,6 +343,59 @@ func (b Board) CurrentPlayer() *Player {
 	return b.Players[b.RemainingPlayers[b.RemainingPlayerIndex]]
 }
 
+// getAccessibleNeighbors returns the tile neighbors that can be accessed.
+func (b Board) getAccessibleNeighbors(line, row int) Coordinates {
```

Comment:
As the board.go file is becoming too large, you may find it easier to move these new methods to another file.